rio_tiler/io/cogeo.py

```diff
@@ -477,10 +398,65 @@ def point(
         if expression:
             blocks = expression.lower().split(",")
             bands = [f"b{bidx}" for bidx in indexes]
-            point = apply_expression(blocks, bands, point).tolist()
+            point = apply_expression(blocks, bands, numpy.array(point)).tolist()
 
         return point
 
+    def tile(
```

Comment:
Why did you reorder this method? Seems substantially similar to the previous `COGReader.tile` method, but since it's moved there's no easy way to see the actual diff.

Reply:
let me undo this